### DISCUSSION TITLE
R5.0 Scope Items Missing in the Microsite

### DISCUSSION THREAD

**Discussion by anunagaraj** [Sunbird-Ed/Community#185] [General Discussion]
@vijiurs @Preethi-Sl Below tickets signed off from ML team for R5.0.0 is not updated in the Release details in Microsite.
SB-28247
SB-28248
SB-30327
SB-29913
RRHE-37
RRHE-32
SB-23100
SB-30386
SB-30495
SB-28305
Please update the Microsite

CC @ShailandraOjha @chitranshu-keshav @luckynvdu @sharathkashyap

**Comment by luckynvdu** [comment]
@anunagaraj The items can be found in the release note.
https://app.gitbook.com/o/-Mi9QwJlsfb7xuxTBc0J/s/-MkgPDmvKwE_DgYJbvPS/use/releases/release-notes/release-4.10.0

**Reply by anunagaraj** [reply]
@luckynvdu The Below tickets which are signed off by ML team is missing in the above release notes as well
SB-29913
RRHE-32
SB-23100
SB-30386
SB-30495
SB-28305
Kindly get the release notes updated.

CC @ShailandraOjha @chitranshu-keshav

**Comment by anunagaraj** [comment]
@luckynvdu @anisha147 Please provide an update on the above missing ticket id's in the release notes

CC @ShailandraOjha @chitranshu-keshav

**Reply by luckynvdu** [reply]
@anisha147 Could you update the same? and update here.

**Reply by ShailandraOjha** [reply]
Hi @luckynvdu the scope for the release should be updated in the released notes as well as in the sunbird ed planned items for that release.
The changes have to be made in both places.
cc: @chitranshu-keshav

**Reply by luckynvdu** [reply]
@ShailandraOjha We have asked the ML team to update the same as there has been some tagging issue in the link to planned items 

@Prateek-slokam

**Reply by ShailandraOjha** [reply]
@Prateek-slokam and @anisha147 Please confirm when it is updated to close the thread. Please.
